geometry_msgs/Point32 loc
string event